cost-analyzer/values.yaml

```
@@ -1871,7 +1871,7 @@ prometheus:
     ##
     image:
       repository: prom/pushgateway
-      tag: v1.8.0
+      tag: v1.8.2
```

```suggestion
      tag: v1.8.0
```
This is wrong. There is no `v1.8.2` tag for pushgateway.